File: backend/ngts/src/main/java/com/three/ngts/Service/TicketAddressService.java
package com.three.ngts.Service;

import com.three.ngts.Entity.TicketAddress;
import com.three.ngts.Repo.TicketAddressRepo;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;

@CrossOrigin(origins = "http://localhost:3000")
@RestController
@RequestMapping("/api/")
public class TicketAddressService {
    @Autowired
    private TicketAddressRepo ticketAddressRepo;

    @PostMapping("/ticketAddresses/insert")
    public TicketAddress insert(@RequestBody TicketAddress ticketAddress) {
        return ticketAddressRepo.save(ticketAddress);
    }
}

File: frontend/next-generation-ticketing-system/src/Service/TicketAddressService.ts
import axios from "axios";

const USER_API_BASE_URL = "http://localhost:8080/api/ticketAddresses";

class TicketAddressService {
  insert(ticketAddress: any) {
    return axios.post(USER_API_BASE_URL + "/insert", ticketAddress);
  }
}

export default new TicketAddressService();
